package.json

```diff
@@ -17,7 +17,8 @@
     "start": "npm run develop",
     "serve": "gatsby serve",
     "lint": "eslint ./src --fix",
-    "pretest": "eslint --ignore-path .gitignore ."
+    "pretest": "eslint --ignore-path .gitignore .",
+    "preload-fonts": "gatsby-preload-fonts"

```

Comment:
How is this script going to be used?

Reply:
`npm run preload-fonts`

Reply:
So, use this before `npm run build`?

Reply:
@Jashpatel1 ^^

Reply:
It has to be run only when fonts are changed.